thunder/tests/test_jit_general.py

```diff
@@ -685,10 +684,6 @@ def test_litgpt_variants(name, device):
         torch.testing.assert_close(param1.grad, param2.grad, rtol=1e-2, atol=1e-2)
 
 
-@pytest.mark.skipif(
-    version_between(torch.__version__, min_ver="2.5.0a0", max_ver="2.5.0a99"),
-    reason="https://github.com/pytorch/pytorch/issues/129579",
-)
```

Comment:
I think, sadly, we need to keep this one with a link to https://github.com/Lightning-AI/lightning-thunder/issues/669 . Apologies for mixing this in here with a misleading comment.